crates/sui-core/src/consensus_validator.rs

```diff
@@ -123,16 +97,17 @@ impl TransactionValidator for SuiTxValidator {
                 | ConsensusTransactionKind::CapabilityNotification(_) => {}
             }
         }
-        // verify the user transaction signatures as a batch
-        obligation
-            .verify_all()
```

Comment:
what about the checkpoints?

Reply:
oh boy, good catch - i didn't notice those were using the same batch. yikes 😨 😨 😨

Reply:
fixed.